Source/CombatExtended/CombatExtended/Comps/CompInventory.cs

```diff
@@ -33,7 +36,21 @@ public CompProperties_Inventory Props
                 return (CompProperties_Inventory)props;
             }
         }
-
+        public bool UpdatingLoadout
+        {
+            get
+            {
+                return GenTicks.TicksGame - updatingLoadoutAge < 5000 && updatingLoadout;
```

Comment:
Magic constant, would probably be better to set it as a const up above with a sensible name.  Also, if the length of time is meaningful, and supposed to be something like every 83 seconds, calculating it from GenTime makes it more explicit how long it is. Or write the duration in human-readable format in a comment.